lib/task-data/tasks/node-sleep.js

```diff
@@ -0,0 +1,17 @@
+// Copyright 2015, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Node sleep',
+    injectableName: 'Task.Node.Sleep',
+    implementsTask: 'Task.Base.Linux.Commands',
+    options: {
+        duration: 10, //in seconds
+        commands: ['sudo sleep {{options.duration}}']
```

Comment:
Why use the micro kernel sleep? isn't same to delay in RackHD. The bluebird has function `Promise.delay` to do delay in async mode.